test/website-nanostores/README.md

```diff
@@ -0,0 +1,47 @@
+# Astro Starter Kit: Minimal
```

Comment:
Starter Kit should be cleaned after. For instance, we don’t need these docs anymore.